R/commonFunctionsAcceptSampling.R

```diff
@@ -451,7 +451,7 @@ getAOQCurve <- function(jaspContainer, pos, depend_vars, df_plan, options, type,
   xBreaks <- jaspGraphs::getPrettyAxisBreaks(c(min(df_plan$PD), max(df_plan$PD)))
   yBreaks <- jaspGraphs::getPrettyAxisBreaks(c(min(df_plan$AOQ), 1.2*aoql))
   plt <- ggplot2::ggplot(data = df_plan, ggplot2::aes(x = PD, y = AOQ)) +
-         ggplot2::geom_point(colour = "black", shape = 19) + ggplot2::labs(x = "(Incoming) Proportion non-conforming", y = "Average Outgoing Quality") +
+         ggplot2::geom_point(colour = "black", shape = 19) + ggplot2::labs(x = "(Incoming) Proportion Non-conforming", y = "Average Outgoing Quality") +
```

Comment:
so this is fine but these also need
```suggestion
         ggplot2::geom_point(colour = "black", shape = 19) + ggplot2::labs(x = gettext("(Incoming) Proportion Non-conforming"), y = gettext("Average Outgoing Quality")) +
```
same for the other plot labels.

Reply:
Added it to the labels. Thanks for pointing out!